src/components/socialLogin/tests/FaceBookActions.test.js

```diff
@@ -0,0 +1,45 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import moxios from 'moxios';
+import loginWithFb from '../../../actions/facebookActions';
```

Comment:
Read the `Component Naming` section here https://github.com/airbnb/javascript/tree/master/react